source/dubregistry/mongodb.d

```diff
@@ -0,0 +1,24 @@
+module dubregistry.mongodb;
+
+import vibe.db.mongo.client : MongoClient;
+import vibe.db.mongo.mongo : connectMongoDB;
+import vibe.db.mongo.settings : MongoClientSettings, MongoAuthMechanism, parseMongoDBUrl;
+
+MongoClientSettings mongoSettings;
+string databaseName = "vpmreg";
+
+shared static this()
```

Comment:
To avoid another `shared static this()`, maybe lazily initialize settings or just parse them everytime connectMongoDB is called (once usually).